cmake/legacy.cmake

```diff
@@ -6,6 +6,50 @@
 #                  copied verbatim in the file "LICENSE"                       #
 ################################################################################
 cmake_minimum_required(VERSION 3.19...3.28 FATAL_ERROR)
+cmake_policy(VERSION 3.19...3.28)
+
+
+if(APPLE)
+  execute_process(COMMAND brew --prefix OUTPUT_VARIABLE brew_prefix)
+  set(LibLZMA_ROOT ${brew_prefix})
+  set(ZSTD_ROOT ${brew_prefix})
+endif()
```

Comment:
Move this please to the `FairSoftConfig.cmake` file (which is mostly needed on macs right now). The idea is that the only this file adds a dependency to `brew` and not the general `cmake/legacy.cmake` file.

Reply:
Fixed with the last commit.